test/test_losses.py

```diff
@@ -0,0 +1,229 @@
+import pytest
+import torch
+import torch.nn.functional as F
+from common_utils import cpu_and_gpu
+from torchvision import ops
+
+
```

Comment:
Making a class, inheriting and the calling method is also possible. For now is this fine?